reactive/filebeat.py

```diff
@@ -190,6 +191,92 @@ def check_filebeat_repo():
     remove_state('filebeat.repo.changed')
 
 
+@when('apt.installed.filebeat')
+@when_any('config.changed.ignore_older', 'config.changed.clean_inactive',
+          'config.changed.scan_frequency')
```

Comment:
are you sure this prevents incorrect values from being rendered by the template? Incorrect values should never make it on disk

Reply:
Hey @aieri, I've been doing some testing and this does not work, no. If I change the config option values via `juju config` the sanity checks work as expected but if I pass the options with invalid values via the bundle the sanity checks never get executed. I've tried various workarounds and flags but nothing has worked. I'm awaiting feedback from Juju SMEs but any suggestions you may have would be very appreciated.

Reply:
this is now resolved, I've added a call to `sanity_check()` to the `render_filebeat_template()` function and if that fails the unit goes into blocked state. The config options are still getting rendered into the file, but in any case we'd need manual intervention to fix an invalid value so IMHO this should be fine.